### DISCUSSION TITLE
Filters

### DISCUSSION THREAD

**Discussion by Jan-Bujnosek** [musescore/MuseScore#12066] [Ideas]
Hi, i love your software, but one thing that i miss from Sibelius is filters, like: "Select all dynamics", "Select all ligatures". You could do it in a selection or in the whole piece.

**Comment by MarcSabatella** [comment]
You can select all dynamics - or all elements of any given type - by right-clicking one then Select / All Similar Elements.  There are also more complex selections possible using the other options in the Select menu.  You can also do *exclude* elements from an existing range selection, using View / Selection Filter.
